docs/FAQ/installing/installing.md

```diff
@@ -0,0 +1,40 @@
+# Installing
+
+
+**What knowledge do I need to install this?**
+
+Depending on the installation way, less specific skill is necessary. For instance, in installation using Docker just follow the tutorial [here](https://docs.shellhub.io/self-hosted/deploying) and that's it! You have the ShellHub on your device.
```

Comment:
Here we need to make clear if user need to use the agent, he/she can use the cloud. The self-hosted install is for the server part. 

Also, this seems to better fit after **Do I need a ShellHub server or can I use the ShellHub Cloud in shellhub.io to manage my devices?** question.